LambdaEngine/Source/Rendering/RenderGraphParser.cpp

```diff
@@ -120,6 +120,7 @@ namespace LambdaEngine
 		}
 
 		//Loop Through each Render Stage in Order and create synchronization stages
+		TSet<String> resourceNamesActuallyUsed;
```

Comment:
I only see strings being inserted into the set; it's never read from. Remove? Also, use an `unordered_set` unless you want the elements sorted.